vital/types/geo_point.cxx

```diff
@@ -40,91 +40,102 @@
 #include <stdexcept>
 
 namespace kwiver {
-namespace vital {
-
-using geo_raw_point_t = geo_point::geo_raw_point_t;
-
-// ----------------------------------------------------------------------------
-geo_point::
-geo_point()
-  : m_original_crs{ -1 }
-{ }
-
-// ----------------------------------------------------------------------------
-geo_point::
-geo_point( geo_raw_point_t const& point, int crs )
-  : m_original_crs( crs )
-{
-  m_loc.insert( std::make_pair( crs, point ) );
-}
-
-// ----------------------------------------------------------------------------
-bool geo_point
-::is_empty() const
-{
-  return m_loc.empty();
-}
-
-// ----------------------------------------------------------------------------
-geo_raw_point_t geo_point
-::location() const
-{
-  return m_loc.at( m_original_crs );
-}
-
-// ----------------------------------------------------------------------------
-int geo_point
-::crs() const
-{
-  return m_original_crs;
-}
-
-// ----------------------------------------------------------------------------
-geo_raw_point_t geo_point
-::location( int crs ) const
-{
-  auto const i = m_loc.find( crs );
-  if ( i == m_loc.end() )
-  {
-    auto const p = geo_conv( location(), m_original_crs, crs );
-    m_loc.emplace( crs, p );
-    return p;
-  }
+  namespace vital {
+
+    using geo_raw_point_t = geo_point::geo_raw_point_t;
```

Comment:
I know KWIVER style is not clearly defined, but I don't think other .cxx files have indentation for everything in the namespace scope.

Reply:
Ok, will fix

Reply:
> I don't think other .cxx files have indentation for everything in the namespace scope.

They don't.
(Actually, I would argue that the style *is* fairly well specified... it just isn't well *followed* :wink:.)